databricks_connector.py

```diff
@@ -0,0 +1,671 @@
+# File: databricks_connector.py
+#
+# Copyright (c) 2023 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+#
+
+import json
+import sys
+import traceback
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import phantom.app as phantom
+import requests
+from phantom.action_result import ActionResult
+from phantom.base_connector import BaseConnector
+
+import databricks_consts as consts
+
+deps_path = Path(__file__).parent.joinpath("dependencies")
+sys.path.insert(0, deps_path.as_posix())
+
+from databricks.sdk import WorkspaceClient  # noqa
```

Comment:
The `noqa` suppresses warnings about import statements not being at the top of the file. This is needed for the Google workaround, but since I'm removing that I'll remove them too.